src/main/java/edu/kis/powp/jobs2d/drivers/ScaledLineDriver.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.transformations.DriverLevelTransformation;
+
+/**
+ * Line adapter - Job2dDriver with DrawPanelController object.
+ */
+public class ScaledLineDriver implements Job2dDriver {
```

Comment:
It is not only about Scaling and it does not need to be about lines only. Your goal should be, e.g., Driver2dTransformationsDecorator.